Scripts/Ports/python3/vcpkg-cmake-wrapper.cmake

```diff
@@ -59,7 +59,7 @@ if(_PythonFinder_WantLibs)
         find_program(
             @PythonFinder_PREFIX@_EXECUTABLE
             NAMES "python" "python@PYTHON_VERSION_MAJOR@.@PYTHON_VERSION_MINOR@"
-            PATHS "${_VCPKG_INSTALLED_DIR}/${VCPKG_TARGET_TRIPLET}/tools/python3"
+            PATHS "${_VCPKG_INSTALLED_DIR}/${VCPKG_HOST_TRIPLET}/tools/python3"

```

Comment:
Actually, now that I'm thinking about it... We need to bump the port-version in vcpkg.json to ensure that this gets deployed.

Reply:
Good catch! Pushed a fix